sphinx/util/__init__.py

```diff
@@ -660,25 +660,30 @@ def old_status_iterator(iterable: Iterable, summary: str, color: str = "darkgree
         logger.info('')
 
 
+def log_status_message(item: str, summary: str, color: str, progress: int,
+                       length: int, verbosity: int = 0) -> None:
+    s = '{}[{:3d}%] {}'.format(summary, (100 * progress) // length,
+                               colorize(color, item))
+    if verbosity:
+        s += '\n'
+    else:
+        s = term_width_line(s)
+    logger.info(s, nonl=True)
```

Comment:
I think the original code is a bit odd because output "\n" with `nonl=True` option. So it's better to output without the option in verbose mode:
```suggestion
    if verbosity:
        logger.info(s)
    else:
        logger.info(term_width_line(s), nonl=True)
```